applications/connecting_applications_to_services/projecting-binding-data.adoc

```diff
@@ -0,0 +1,23 @@
+[id="projecting-binding-data"]
+= Projecting binding data
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: projecting-binding-data
+
+toc::[]
+
+This section provides information on how you can consume the binding data.
+
+== Consumption of binding data
+For accessing and consuming the binding data from a workload, this data has to be projected into the workload from a backing service. After the backing service exposes the binding data, {servicebinding-title} Operator automatically projects this set of data into the workload in the following methods:
+
+. By default, as files.
+. As environment variables, after you configure the `.spec.bindAsFiles` parameter from the `ServiceBinding` resource.
+
+include::modules/sbo-configuration-of-directory-path-to-project-binding-data.adoc[leveloffset=+1]
+include::modules/sbo-projecting-the-binding-data.adoc[leveloffset=+1]
+
+== Additional resources
+//Commenting out the link to avoid Build errors. The entire SBO section is new with topics. Links will work fine when all the SBO doc PRs will be merged.
+
+* For more information on how to expose binding data, see Exposing binding data from a service.
```

Comment:
```suggestion
* Exposing binding data from a service
```
For minimalism, you can remove the surrounding text and simply provide the link.

Reply:
Will implement this.